schema/aggregations/filter_artworks_aggregation.js

```diff
@@ -24,17 +24,11 @@ export const ArtworksAggregation = new GraphQLEnumType({
     MEDIUM: {
       value: 'medium',
     },
-    FOR_SALE: {
-      value: 'for_sale',
-    },
     GALLERY: {
-      value: 'partner_id',
```

Comment:
I don't understand why this is `partner_id`. That doesn't do anything. The correct form is just `aggregations[]=gallery`